packages/chart/src/components/EditorPanel/EditorPanel.tsx

```diff
@@ -2642,6 +2642,18 @@ const EditorPanel = () => {
                                 </label>
                               )}
                             </li>
+                            <li>
```

Comment:
I think you might want to add visualizationType === 'Pie' here. It doesn't look like its used on any other chart types.